src/cells/fsm.mjs

```diff
@@ -189,6 +189,7 @@ export const FSMView = BoxView.extend({
     events: {
         "click foreignObject.tooltip": "stopprop",
         "mousedown foreignObject.tooltip": "stopprop",
+        "touchstart foreignObject.tooltip": "stopprop", // make sure the input recieve focus
```

Comment:
The word is "receive". Could you correct this (in all relevant places)?

Reply:
Ooops. Done.